lms/envs/common.py

```diff
@@ -2295,3 +2295,6 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# This is to check the domain in case of preview.
+PREVIEW_DOMAIN = 'preview'
```

Comment:
I see what you are saying now. Everything works as expected with `preview` but changing it to anything breaks the preview functionality i.e. it does not show un-published changes. right?